.github/workflows/packaging.yml

```diff
@@ -0,0 +1,105 @@
+name: packaging
+
+on: [push, pull_request]
+
+jobs:
+  # Run not only on tags, otherwise dependent job will skip.
+  version-check:
+    # Skip pull request job when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        uses: tarantool/actions/check-module-version@master
+        with:
+          module-name: 'avro-schema'
+
+  package:
+    # Skip pull request jobs when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    # Packaging for CentOS 7 does not work with other versions, see:
+    # https://github.com/packpack/packpack/issues/145
+    runs-on: ubuntu-20.04
+    needs: version-check
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          # See https://github.com/tarantool/avro-schema/issues/148
```

Comment:
Look at https://github.com/tarantool/queue/commit/769bc725f8fe60714996a6bde5cf6c260d9b92fd

Reply:
Excluded only Fedora 33+, added new info to ticket (it seems that we may need to rework cmake here too)

Reply:
For reasons yet unknown, Fedora 31 tries to install Tarantool 2.1.2 and fails to find corresponding devel package. Since Fedora 31 EOLed 2020-11-24, I do not support it here too.